cmd/bootstrap/run/execution_state.go

```diff
@@ -41,15 +42,23 @@ func GenerateExecutionState(
 	if err != nil {
 		return flow.DummyStateCommitment, err
 	}
-	defer func() {
-		<-diskWal.Done()
-	}()
 
 	ledgerStorage, err := ledger.NewLedger(diskWal, 100, metricsCollector, zerolog.Nop(), ledger.DefaultPathFinderVersion)
 	if err != nil {
 		return flow.DummyStateCommitment, err
 	}
 
+	compactor, err := complete.NewCompactor(ledgerStorage, diskWal, zerolog.Nop(), 100, 1_000_000, 1)
```

Comment:
when cleaning up please add constants for these default values.

Reply:
and also we should pass the logger here as well in my view.

Reply:
Good catch!  Fixed in commit c5455bd07435c7664565e5c169f6e1d601ecf827.